Sources/TensorFlow/Operators/Basic.swift

```diff
@@ -768,34 +768,66 @@ extension Tensor where Scalar: TensorFlowFloatingPoint {
 // Padding
 //===------------------------------------------------------------------------------------------===//
 
+
 public extension Tensor where Scalar: Numeric {
-    /// Returns a padded tensor according to the specified padding sizes.
+    /// Padding modes:
+    ///  * `constant` - pad with constant value.
+    ///  * `reflect` - mirror values along padding dimensions, excluding the edge value.
+    ///  * `symmetric` - mirror values along padding dimensions, including the edge value.
+    enum PaddingMode {
+        case constant(Scalar)
+        case reflect
+        case symmetric
+    }
+
+    /// Returns a tensor padded with constant according to the specified padding sizes.
     @inlinable
-    @differentiable(wrt: self, vjp: _vjpPadded(forSizes:with:) where Scalar: TensorFlowFloatingPoint)
+    @differentiable(wrt: self where Scalar: TensorFlowFloatingPoint)
     func padded(forSizes sizes: [(before: Int, after: Int)], with value: Scalar = 0) -> Tensor {
+        return self.padded(forSizes: sizes, with: .constant(value))
```

Comment:
You can omit `self.` and `return`.
```suggestion
        padded(forSizes: sizes, with: .constant(value))
```